src/wait-for-angular.js

```diff
@@ -0,0 +1,42 @@
+/*global Promise*/
+import { ClientFunction } from 'testcafe';
+
+export default ClientFunction(ms => {
+    return new Promise((resolve, reject) => {
+        let pingIntervalId  = null;
+        let pingTimeoutId   = null;
+        const WAIT_TIMEOUT  = ms || 10000;
+        const PING_INTERVAL = 100;
+
+        const clearTimeouts                   = () => {
+            window.clearTimeout(pingTimeoutId);
+            window.clearInterval(pingIntervalId);
+        };
+        const isThereAngularInDevelopmentMode = () => {
+            if (window.ng && typeof window.ng.probe === 'function' &&
+                typeof window.getAllAngularRootElements === 'function') {
+                const rootElements          = window.getAllAngularRootElements();
+                const firstRootDebugElement = rootElements &&
+                                              rootElements.length ? window.ng.probe(rootElements[0]) : null;
+
+                return !!(firstRootDebugElement && firstRootDebugElement.injector);
+            }
+
+            return false;
+        };
+        const check                           = () => {
```

Comment:
separate different multi-line var declarations with empty line, it'll improve readability.